### ISSUE TITLE
Hero text

### ISSUE THREAD

**Issue by BrandMasterG** [LynnUniversity/lynnedu#31]
I do not like how the hero text drops down below the hero images on mobile. Particularly bad right on home landing where image and play button is below video image area. For others I can easily go back in and cut or rework the head copy where necessary for better fit.

**Comment by bchanceylynn** [comment]
Hi Gina, 

 Is the suggestion to shorten the copy or create a better solution to show the copy?

**Comment by BrandMasterG** [comment]
shorten. I am happy to work on this or help

**Comment by bchanceylynn** [comment]
Okay. Let’s take a look together when you have time so I can make sure we are on the same page.

**Comment by BrandMasterG** [comment]
YES! Want to spend some time today maybe after the call? If not please feel free to call a meeting for us.
